htdocs/compta/recap-compta.php

```diff
@@ -224,7 +245,11 @@
 			// Display array
 			foreach($TData as $data) {
 				
-				print '<tr class="oddeven">';
+				$html_id = '';
+				if (!empty($data['fk_facture'])) $html_id = 'facid-'.$data['fk_facture'];
+				elseif (!empty($data['fk_paiement'])) $html_id = 'payid-'.$data['fk_paiement'];
+				
+				print '<tr id="'.$html_id.'" class="oddeven">';
```

Comment:
This may create duplicate id with same value. This is forbidden by HTML5. May be ou can move this in class ?